libraries/chain/include/eosio/chain/webassembly/interface.hpp

```diff
@@ -306,6 +306,17 @@ namespace webassembly {
          */
          void set_privileged(account_name account, bool is_priv);
 
+         /**
+          * Set the resource payer for the transaction.
+          *
+          * @ingroup privileged
+          *
+          * @param payer - name of the account who is paying for the transaction.
+          * @param max_net - max amount of net resource to be paid for the transaction.
+          * @param max_cpu - max amount of cpu resource to be paid for the transaction.
```

Comment:
Make it clear if this is microseconds or milliseconds. I believe the requirements are to support microseconds unlike the current transaction max_cpu_usage_ms which is milliseconds.

Reply:
Done